test/hts-precompile/utils.js

```diff
@@ -198,6 +214,28 @@ class Utils {
       expect(e.code).to.eq(code);
     }
   }
+
+  static async createClient(operatorId, operatorKey, hederaNetwork = { "127.0.0.1:50211": new AccountId(3) }, mirrorNode = "127.0.0.1:5600") {
```

Comment:
Was going to suggest to add a config file to pick up client details but for now maybe rename to `createLocalSDKClient` so it's explicit